README.md

```diff
@@ -96,20 +85,14 @@ Contributions are welcome and encouraged.  You can contribute in many ways:
 
 The best place to start is the IRC channel on freenode (see above), drop in, say hi. Next place is the github issue tracker:
 
-https://github.com/cleanflight/cleanflight/issues
-https://github.com/cleanflight/cleanflight-configurator/issues
+https://github.com/iNavFlight/inav/issues
+https://github.com/iNavFlight/inav-configurator/issues
 
 Before creating new issues please check to see if there is an existing one, search first otherwise you waste peoples time when they could be coding instead!
 
 ## Developers
 
-Please refer to the development section in the [docs/development](https://github.com/cleanflight/cleanflight/tree/master/docs/development) folder.
-
-TravisCI is used to run automatic builds
-
-https://travis-ci.org/cleanflight/cleanflight
-
-[![Build Status](https://travis-ci.org/cleanflight/cleanflight.svg?branch=master)](https://travis-ci.org/cleanflight/cleanflight)
+Please refer to the development section in the [docs/development](https://github.com/iNavFlight/inav-configurator/tree/master/docs/development) folder.
```

Comment:
1. It should not be necessary for us to pimp Hydra's irc windows's client 
2. perhaps we should mention the RCG thread, as the irc channel is not exactly a hub of activity